packages/slave/slave.js

```diff
@@ -1296,6 +1301,7 @@ class Slave extends libLink.Link {
 		libLink.messages.instanceStatusChanged.send(this, {
 			instance_id,
 			status: instanceConnection ? instanceConnection.status : "stopped",
+			game_port: null,
```

Comment:
This will cause the game_port to always be notified to the master as null after a reconnect, which is definitively not right.